test/wallet-test.js

```diff
@@ -245,6 +245,12 @@ describe('Wallet', function() {
     consensus.COINBASE_MATURITY = 0;
   });
 
+  it('should get dummy chain info (from NullClient)', async () => {
+    const getInfo = await wdb.client.getInfo();
```

Comment:
`await` unneeded

Reply:
:+1: In this case no you're right thanks